docs/strategy_analysis_example.md

```diff
@@ -250,3 +250,75 @@ fig.show()
 ```
 
 Feel free to submit an issue or Pull Request enhancing this document if you would like to share ideas on how to best analyze the data.
+
+## Analyse the buy/entry and sell/exit tags
+
+It can be helpful to understand how a strategy behaves according to the buy/entry tags used to
+mark up different buy conditions. You might want to see more complex statistics about each buy and
+sell condition above those provided by the default backtesting output. You may also want to
+determine indicator values on the signal candle that resulted in a trade opening.
+
+We first need to enable the exporting of trades from backtesting:
+
+```
```

Comment:
```suggestion
``` bash
```

Reply:
Fixed in [b3cb722](https://github.com/freqtrade/freqtrade/pull/6543/commits/b3cb7226467c359b0748cb4866f8f8fcfa649a53)